lib/resources/kernel_module.rb

```diff
@@ -37,7 +38,7 @@ def loaded?
     end
 
     def version
-      if inspec.os[:name] == 'centos' && inspec.os[:release].to_i == 5
+      if inspec.os.family == 'redhat'
```

Comment:
@postgred it is better to use `inspec.os.redhat?`. we try to avoid `inspec.os.family`. This is inherited from Serverspec. We are going to prefer the name or the helper method.